internal/config/main_config.go

```diff
@@ -64,6 +64,9 @@ var config struct {
 		ClientID       string
 		ClientSecret   string
 	}
+	GCP struct {
+		Credentials string
+	}
```

Comment:
Nitpick: I would add a validator, I am pretty sure people will try to put some invalid inputs into this setting and not JSON. We do have a validate function already, just try to parse it with JSON and panic on errors or something.